source/aggregation-pipeline-builder.txt

```diff
@@ -172,6 +176,70 @@ your chosen view mode:
             
             |compass-short| returns your results in the document view. 
 
+   .. tab:: Stage Wizard 
+      :tabid: wizard 
+
+      .. procedure::
+         :style: normal
+         
+         .. step:: Select the :guilabel:`Stages` view.
+            
+            In the aggregation pipeline pane, ensure the 
+            :guilabel:`{} Stages` toggle switch is selected.
+
+            .. figure:: /images/compass/agg-builder-stageview-toggle-on.png
+               :figwidth: 300 px
+               :alt: Toggle on stage view mode 
+
+         .. step:: Open the :guilabel:`Stage Wizard` card. 
+            
+            To the right of the view mode toggle, click the funnel icon to open 
+            the :guilabel:`Stage Wizard` card.
+
+            .. figure:: /images/compass/agg-builder-stage-wizard.png
+               :figwidth: 300 px
+               :alt: Click the icon to the right of the view mode toggle.
+
+         .. step:: Select an aggregation pipeline stage use case.
+
+            On the :guilabel:`Stage Wizard` card, select the aggregation 
+            pipeline stage use case for the first stage of the pipeline.
+            |compass-short| populates the the stage card with a form that 
+            corresponds with the selected aggregation pipeline stage.
+
+         .. step:: Fill in your pipeline stage. 
+
+            Fill in the form for your selected stage and click 
+            :guilabel:`Apply`. After you click :guilabel:`Apply`, the form will 
+            turns into a stage card that you can edit in Stage View Mode, Focus 
+            Mode, or Text View Mode. 
+
+            |compass-short| populates the Stage Output with up to ten sample 
+            output documents.
+
+            .. note:: 
+
+               You cannot edit an existing stage through the Stage Wizard. The 
+               Stage Wizard can only add new stages. To edit an existing stage, 
+               use Stage View Mode, Focus Mode, or Text View Mode.
```

Comment:
Suggest moving this up, right after line 100 where we talk about the options for building out a pipeline

Reply:
Leaning towards keeping the note here bc the blurbs at the top of the "Create an Aggregation Pipeline" are meant to be a sort of "at-a-glance" summary of each view mode/feature with specifics/limitations included in the corresponding task.